test/e2e/positive.go

```diff
@@ -50,4 +65,140 @@ var _ = ginkgo.Describe("[PositiveFlow] Deployer validation", func() {
 			gomega.Expect(vo.Errors).To(gomega.BeEmpty())
 		})
 	})
+
+	ginkgo.Context("with a running cluster without any components", func() {
+		ginkgo.BeforeEach(func() {
+			err := deploy()
+			gomega.Expect(err).ToNot(gomega.HaveOccurred())
+		})
+
+		ginkgo.AfterEach(func() {
+			err := remove()
+			gomega.Expect(err).ToNot(gomega.HaveOccurred())
+		})
+
+		ginkgo.It("should perform overall deployment and verify all pods are running", func() {
+			ginkgo.By("checking that resource-topology-exporter pod is running")
+			mf, err := rte.GetManifests()
+			gomega.Expect(err).ToNot(gomega.HaveOccurred())
+			mf = mf.UpdateNamespace()
+
+			gomega.Eventually(func() error {
+				pod, err := getPodByRegex(fmt.Sprintf("%s-*", mf.DaemonSet.Name))
```

Comment:
this work under current CI on which we have exactly one pod because we will have exactly one worker due to resource constraints. But this should be a loop and collect all the pods, be them 1, 2 or 20. Could be fixed in a later PR.